hardhat.config.js

```diff
@@ -57,20 +57,58 @@ for (const f of fs.readdirSync(path.join(__dirname, 'hardhat'))) {
 const withOptimizations =
   argv.enableGasReport || argv.compileMode === 'production';
 
+const requestFaucet = async (wallet, amount) => {
+  // TODO: create a wrapper
+  if (process.env.USE_FAUCET === 'true') {
+    console.log('Skipping faucet request');
```

Comment:
it's not a mistake? If USE_FAUCET=true this code skip request to faucet, but code below create a response for it

Reply:
Yes, a mistake
Config is a typescript file

Reply:
fixed